go/ql/lib/semmle/go/security/CommandInjectionCustomizations.qll

```diff
@@ -29,8 +29,13 @@ module CommandInjection {
    */
   abstract class Sanitizer extends DataFlow::Node { }
 
+  /**
+   * DEPRECATED: Use `RemoteFlowSource` or `Source` instead.
+   */
+  deprecated class UntrustedFlowAsSource = RemoteFlowAsSource;
```

Comment:
Does this comment make sense, given that `RemoteFlowAsSource` is marked as `private` here?

Reply:
The comment doesn't mention `RemoteFlowAsSource`, it mentions `RemoteFlowSource`.

Reply:
I genuinely did not notice that when reading any of the comments I flagged up! My bad then, the comments are all fine then.